tests/language-feature/capability/intrinsic-texture-ignore-capability.slang

```diff
@@ -0,0 +1,498 @@
+#version 450 core
```

Comment:
What is the purpose of this test?

Reply:
You should write it in the test file as a comment.